conduit/plugins/tools/testutil/mockinit.go

```diff
@@ -13,6 +13,11 @@ func (m *MockInitProvider) GetGenesis() *sdk.Genesis {
 	return m.Genesis
 }
 
+// SetGenesis updates the genesis block in the init provider
+func (m *MockInitProvider) SetGenesis(genesis *sdk.Genesis) {
```

Comment:
IIRC This implementation is identical to the regular InitProvider but there is a different constructor function. Then there is a separate interface and implementation for the regular InitProvider. There was some complication with circular dependencies when I tried to consolidate them all.

Reply:
Removed it since they're the same. Left the interface in place so we can implement a new type if we need to differ the behavior ever.